src/plugins/line/README.md

```diff
@@ -8,5 +8,35 @@
 
 ## Introduction ##
 
-This plug-in is designed to save each line from input as a key. They
-keys are in an array.
+This plugin is designed to save each line from input as a key. The
+keys are stored in an array. The key names are determined by the 
+line number such as `#01` or `#12` and the value of each key 
+is the infromation stored in that line of the file.  
+
+For instance, consider the following file name `.config` where the 
+numbers on the left just represent the line numbers:
+
+    1  setting1 true
+    2  setting2 false
+    3  setting3 1000
+    4  #comment
+    5
+    6
+    7  //some other comment
+    8
+    9  setting4 -1
+    10 
+
+This file would result in the follwing keyset which is being diplayed as
+`name: value`:
+
+    #01: setting1 true
```

Comment:
that changed, its now `#0` to allow arbitrary long files.

Can you describe that?